style.css

```diff
@@ -0,0 +1,364 @@
+* {
+  -webkit-box-sizing: border-box;
```

Comment:
What is the difference between `-webkit-box-sizing:` and `box-sizing:` ?

Reply:
When you use sass, it automatically compiles into css and adds code for other browsers